mlflow/entities/run_info.py

```diff
@@ -18,8 +18,10 @@ def check_run_is_deleted(run_info):
                               .format(run_info.run_id))
 
 
-class numeric_property(property):
-    is_numeric = True
+class attribute(property):
```

Comment:
The word attribute is quite generic, maybe we could just call this `searchable` or something?

Reply:
that makes sense. I like `searchable_attribute` -- this in internal class and not public API, but good for readability.